### PR TITLE
 feat(tasks): Add general and approval task type icons

### PR DESCRIPTION
**Pull request by shahzadaziz**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @shahzadaziz has signed the CLA. Thanks for the pull request!

File: src/icons/tasks/IconTaskApproval.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+// @flow
+import * as React from 'react';
+
+import AccessibleSVG from '../accessible-svg';
+
+type Props = {
+    className?: string,
+    color?: string,
```
Comment (jstoffan) [inline src/icons/tasks/IconTaskApproval.js]: I'm under the impression we intend to style icons via CSS classes exclusively going forward. @tjgupta, can you take a quick look?

File: conf/styleguide.config.js [hunk 2]
```diff
@@ -174,6 +174,14 @@ const allSections = [
                 name: 'States',
                 content: '../src/icons/states/README.md',
             },
+            {
+                name: 'Tasks',
+                components: () => [
```
Comment (jstoffan) [inline conf/styleguide.config.js]: We've started to prefer a single content page for multiple small components. See `name: 'States'` above for an example.

File: src/icons/tasks/IconTaskGeneral.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+type Props = {
+    className?: string,
+    color?: string,
+    height?: number,
+    /** A text-only string describing the icon if it is not purely decorative for accessibility */
+    title?: string,
+    width?: number,
+};
+
+const IconTaskGeneral = ({ className = '', color = '#26C281', height = 30, title, width = 30 }: Props) => (
```
Comment (jstoffan) [inline src/icons/tasks/IconTaskGeneral.js]: It looks like these icons are using different default dimensions and viewbox values than most others. Is that intentional?

File: src/icons/tasks/IconTaskApproval.js (hunk 1 above)
Comment (tjgupta) [reply src/icons/tasks/IconTaskApproval.js]: Yep, I can work with @shahzadaziz to conform this to our two-toned icon style.

File: conf/styleguide.config.js (hunk 2 above)
Comment (booc0mtaco) [reply conf/styleguide.config.js]: @jstoffan when using README.md, that requires ... `require()` for each component, correct? with the individual files, the markdown files seem to infer which component to import.

File: src/icons/tasks/IconTaskGeneral.js (hunk 3 above)
Comment (shahzadaziz) [reply src/icons/tasks/IconTaskGeneral.js]: yes it was intentional.

File: conf/styleguide.config.js (hunk 2 above)
Comment (jstoffan) [reply conf/styleguide.config.js]: Yes, that's true. The READMEs serve a dual purpose as Github documentation and generators for the example site. Having a list of `require` statements at the top of the README seems like a reasonable tradeoff, but if that's a concern, then the other icon READMEs will also need to be reverted to single components. 

@booc0mtaco @tjgupta, can you sync up on the desired strategy here?

**Pull request merged**